orc8r/cloud/configs/orc8r_worker.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file should essentially be a duplicate of orc8r/cloud/configs/state.yml
```

Comment:
If we update orc8r/cloud/configs/state.yml, do we update this file too? (In that case, it might be good to keep a note of that in the other file)

Reply:
I think that this is temporarily the same as the other file. By the end of the milestone, these two flags should be removed from the other file! Maybe I should add a TODO in the other file to make it more clear?

Reply:
if it's a short term thing, then I think we're ok :)